### PR TITLE
Brexit brand author icons

### PR DESCRIPTION
**Pull request by sumeetadur**
screen shot 2016-03-23 at 10 04 50

### CODE REVIEW

File: shared/components/stickies/sticky-item/index.js [hunk 1]
```diff
@@ -3,6 +3,10 @@ import ODate from '../../o-date';
 import ImageContainer from '../../image-container';
 import SaveForLater from '../../save-for-later';
 
+const brandIcons = {
+	'Q0ItMDAwMDkwMA==-QXV0aG9ycw==': 'http://ig.ft.com/_other/ft-author-brand-images/martin-wolf-small.png'
```
Comment (andygnewman) [inline shared/components/stickies/sticky-item/index.js]: You could use the n-article-branding module to get author headshots. https://github.com/Financial-Times/n-article-branding

Comment (callumlocke) [inline shared/components/stickies/sticky-item/index.js]: there was a weeks-long design debate over this :) in the end we were told to use this new style of author images for the Brexit stream. I believe this new style is being considered for Next-wide adoption, but it's experimental-brexit-only for now.

image

**Comment by callumlocke** [comment]
@i-like-robots could you take a look at this?

**Comment by i-like-robots** [comment]
The section cards on the front page have headshots too...

https://github.com/Financial-Times/next-front-page/pull/642

How does this fit in with that?

**Comment by i-like-robots** [comment]
Or I mean to say, can this fit in with that? Do designers need to get in touch with each other?

**Comment by matthew-andrews** [comment]
I agree with matt — this should be done in the same way as the front page.
https://github.com/Financial-Times/n-article-branding

**Comment by sumeetadur** [comment]
@callumlocke

**Comment by callumlocke** [comment]
Looks good to me, if build fixes itself

Sumeet could you paste in a screenshot so matts a and h can see?

**Comment by sumeetadur** [comment]
Thanks. Screenshot added to first comment

**Comment by callumlocke** [comment]
@i-like-robots @matthew-andrews ok to merge now? Design now follows front page

**Comment by callumlocke** [comment]
Oh yeah build still failing. Can't look into it because on phone, but could this be related to the npm debacle?

**Comment by sumeetadur** [comment]
2 errors. 1 is it can't find the heroku app, the other is an error in _brand.scss in n-card:

screen shot 2016-03-23 at 10 16 58

screen shot 2016-03-23 at 10 17 27

**Comment by matthew-andrews** [comment]
don't worry about the heroku app one — it's happening because the build isn't getting far enough to create it

**Comment by sumeetadur** [comment]
Fixed now thanks to the recent update on n-card. @i-like-robots @matthew-andrews Ok to merge?

File: shared/components/stickies/sticky-item/index.js [hunk 2]
```diff
@@ -3,6 +3,17 @@ import ODate from '../../o-date';
 import ImageContainer from '../../image-container';
 import SaveForLater from '../../save-for-later';
 
+// TODO: get these into the headshots repo
```
Comment (matthew-andrews) [inline shared/components/stickies/sticky-item/index.js]: all of these columnists *are* in the headshots repo already…?
https://github.com/Financial-Times/n-article-branding/blob/master/data/headshot-mapping-data.json

Comment (sumeetadur) [inline shared/components/stickies/sticky-item/index.js]: Thanks, forgot to remove that comment :see_no_evil:

**Comment by callumlocke** [comment]
if there's nothing else, I think this is good to go :+1:

File: shared/components/stickies/sticky-item/index.js [hunk 3]
```diff
@@ -3,6 +3,16 @@ import ODate from '../../o-date';
 import ImageContainer from '../../image-container';
 import SaveForLater from '../../save-for-later';
 
+const brandIcons = {
```
Comment (matthew-andrews) [inline shared/components/stickies/sticky-item/index.js]: why can't we use n-article-branding ?

**Comment by matthew-andrews** [comment]
!

**Comment by matthew-andrews** [comment]
why can't we use n-article-branding??

**Comment by matthew-andrews** [comment]
come on guys you're not supposed to merge things until it's been reviewed properly

**Comment by callumlocke** [comment]
sorry, I thought you were done with it as there was just a comment about that comment.

**Comment by matthew-andrews** [comment]
it should use the same logic as the home page

**Comment by sumeetadur** [comment]
sorry, I took Callum's :+1: as a go-ahead. I'll look at the n-article-branding logic now

**Comment by callumlocke** [comment]
My mistake, I told Sumeet to merge. I misunderstood what n-article-branding was and thought it wasn't applicable for our custom use case, but it is. Sumeet is now switching over to use n-article-branding.

**Pull request merged**